python-venv/cmstools/test/cfs_sessions_rc_test/cfs/session.py

```diff
@@ -0,0 +1,247 @@
+#
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+"""
+CFS Session related functions
+"""
+
+import requests
+
+from cmstools.lib.api import request, request_and_check_status
+from cmstools.lib.cfs.defs import (CFS_SESSIONS_URL_TEMPLATE, HTTP_OK, HTTP_NO_CONTENT, HTTP_BAD_REQUEST,
+                                   HTTP_NOT_FOUND, SessionGetWithNameContainsResult)
+from cmstools.lib.api.api import API_REQUEST_TIMEOUT, add_api_auth, SYSTEM_CA_CERTS
```

Comment:
Would it be preferable to be able to import this just from `cmstools.lib.api`, rather than having to know which specific file they are defined in (in case the internals of that module are altered)? Then this can just be combined with the imports on line 31.